hack/pvc-autoscaler-up.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
+# SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+WITH_LPP_RESIZE_SUPPORT=${WITH_LPP_RESIZE_SUPPORT:-true}
```

Comment:
Do we need this? Can't we just use 
```suggestion
WITH_LPP_RESIZE_SUPPORT=true
```

Reply:
If we run the script without using the make target, there's still the `--with-lpp-resize-support` flag. This is actually the one that's also used in the Makefile. 

We can leave it as it is but Isn't it a bit confusing to have two ways to set the same argument?